rest-api/src/test/java/gov/cms/qpp/conversion/api/integration/CpcRestIntegrationTest.java

```diff
@@ -0,0 +1,96 @@
+package gov.cms.qpp.conversion.api.integration;
+
+import gov.cms.qpp.conversion.api.helper.JwtPayloadHelper;
+import gov.cms.qpp.conversion.api.helper.JwtTestHelper;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.context.web.WebAppConfiguration;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+import org.springframework.web.context.WebApplicationContext;
+
+import static org.springframework.security.test.web.servlet.setup.SecurityMockMvcConfigurers.springSecurity;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@SpringBootTest
+@WebAppConfiguration
+@RunWith(SpringRunner.class)
+@TestPropertySource(locations = "classpath:test.properties")
+public class CpcRestIntegrationTest {
```

Comment:
Are positive scenarios possible?

Reply:
I don't think so since positive scenarios require DynamoDB and S3 to be hooked up.  I don't want to create that dependency for these quick-running integration tests.  I decided to leave positive tests to #541.